src/Chart.tsx

```diff
@@ -104,6 +110,49 @@ export const Chart = forwardRef<ChartHandle, ChartProps>(
 			);
 		}
 
+		if (hasBigNumberChildren && toArray(props.children).length != bigNumberChildren.length) {
+			throw new Error(
+				'If passing BigNumber to Chart only the BigNumber will be displayed. All other elements will be ignored'
```

Comment:
Ideally we could throw an error during development if the children being passed are incompatible. That's most likely pretty complicated to put together though. Even though it's best practice to avoid silent failures, I think it would be preferable over having the API allow the users to pass in multiple children only to throw an error. 

Let's make this a console.warn instead.